addon/commands/_private/rdfa-commands/wrap-literal.ts

```diff
@@ -0,0 +1,27 @@
+import { Command } from 'prosemirror-state';
+import { wrapIn } from 'prosemirror-commands';
+import { v4 as uuidv4 } from 'uuid';
+import { findRdfaIdsInSelection } from '@lblod/ember-rdfa-editor/utils/rdfa-utils';
+
+export function wrapLiteral(): Command {
+  return (state, dispatch) => {
+    const childRdfaIds = findRdfaIdsInSelection(state.selection);
+    if (childRdfaIds.size !== 0) {
+      if (dispatch) {
+        console.error('Cannot wrap child rdfa nodes in a literal node');
```

Comment:
that's a good pattern, to notify but not throw if you're running the command without checking :+1: 
We should use it more often